### PR TITLE
Sdk major release 2024

### PR DESCRIPTION
**Pull request by apaparazzi0329**

### CODE REVIEW

File: assistant/src/main/java/com/ibm/watson/assistant/v2/model/BaseEnvironmentOrchestration.java [hunk 1]
```diff
@@ -21,58 +22,8 @@ public class BaseEnvironmentOrchestration extends GenericModel {
   @SerializedName("search_skill_fallback")
   protected Boolean searchSkillFallback;
 
-  /** Builder. */
-  public static class Builder {
```
Comment (kevinkowa) [inline assistant/src/main/java/com/ibm/watson/assistant/v2/model/BaseEnvironmentOrchestration.java:25]: is the builder no longer required here?

File: assistant/src/main/java/com/ibm/watson/assistant/v2/model/BaseEnvironmentReleaseReference.java [hunk 2]
```diff
@@ -19,66 +20,16 @@ public class BaseEnvironmentReleaseReference extends GenericModel {
 
   protected String release;
 
-  /** Builder. */
-  public static class Builder {
```
Comment (kevinkowa) [inline assistant/src/main/java/com/ibm/watson/assistant/v2/model/BaseEnvironmentReleaseReference.java:23]: here too

**Review by kevinkowa — APPROVED** [review]
lgtm just one question on the removed builders

File: assistant/src/main/java/com/ibm/watson/assistant/v2/model/BaseEnvironmentOrchestration.java (hunk 1 above)
Comment (apaparazzi0329) [reply assistant/src/main/java/com/ibm/watson/assistant/v2/model/BaseEnvironmentOrchestration.java:25]: Good eye 👁️. These models are response models so they do not need builders. Not sure why they had builders before but they are not necessary

File: assistant/src/main/java/com/ibm/watson/assistant/v2/model/BaseEnvironmentReleaseReference.java (hunk 2 above)
Comment (apaparazzi0329) [reply assistant/src/main/java/com/ibm/watson/assistant/v2/model/BaseEnvironmentReleaseReference.java:23]: Same as above

**Review by maxresdefault2 — APPROVED** [review]
Reviewed `InputStreamConnectStrategy` and `MessageEventDeserializer`. Both look good to me!

**Pull request merged**